data/kahet/kahet missions.txt

```diff
@@ -2047,3 +2047,365 @@ conversation "end of Patir 2 - on complete"
+			`	Strangely enough, this moon does not show any sign of strip mining, or in fact any digging at all. After operating mining facilities for a hundred thousand years, the surface should show clearly visible signs of such an intensive operation, but there seem to be none.`
+			choice
+				`	(Fly closer to the mining outposts to investigate.)`
+				`	(Move to the target the Remnant gave you.)`
+					goto quarantine
+			`	You pitch down your ship and steer toward the nearest outpost: a long series of short buildings with angled roofs, reminiscent of a solar field in your ship's screen. After coming closer, however, you realize your initial guess was wrong: these are not akin to roofs, but to sails, composed of a pale white textile. A thin layer of dust covers their surface, mostly gathered toward the bottom.`
+			`	You try to come closer, but the compass on your ship suddenly starts spinning randomly! Alarmed, you swiftly reverse course to move away from the facility, and the instrumentation soon returns to normal again after a reboot. The automated outpost is clearly emitting a strong magnetic field, but the connection among all this isn't immediately obvious.`
+			choice
+				`	(Do a close flyby, even if the compass goes haywire.)`
+				`	(Try to search around the facility, keeping a safe distance.)`
+					goto coward
+				`	(Give up and move to the original target.)`
+					goto quarantine
+			`	A brief exploration doesn't turn up anything you hadn't already seen earlier, and all you can do is watch as the mesmerizing flow of Faes'mar in and out of the facility continues undisturbed below you.`
+			label coward
+			`	Just as you're starting to feel that this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of precious metal particles.`
+			`	The overall picture has become clearer: the frequent asteroid impacts on a moon like this raise enough meteorite dust to effectively shroud the planet in an extremely sparse atmosphere, which the Builder machines attract and gather so that they can be transported off-planet. It's a slow and cumbersome process, but it's also one that was able to work for tens of thousands of years without maintenance.`
+			label quarantine
+			`	You record a brief log of your thoughts, and move on to your actual destination. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, the map eventually signals you have reached your destination: a plain, unremarkable plateau extending for kilometers in every direction.`
```

Comment:
```suggestion
			`	You record a brief log of your thoughts, and move on to the target location. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, you eventually reach your destination, unremarkable plateau extending for kilometers in every direction.`
```
It sounds better to keep this more generalized than to specifically say my map is what's telling me I'm at the location.